orc8r/cloud/go/services/bootstrapper/servicers/registration/cloud_registration.go

```diff
@@ -53,7 +56,7 @@ func (c *cloudRegistrationServicer) GetToken(ctx context.Context, request *proto
 		glog.V(2).Infof("could not get tokenInfo for networkID %v and logicalID %v: %v", networkId, logicalId, err)
 	}
 
-	refresh := request.Refresh || tokenInfo == nil || IsExpired(tokenInfo)
+	refresh := (c.refreshOnRequest && request.Refresh) || tokenInfo == nil || IsExpired(tokenInfo)
```

Comment:
@hcgatewood i feel like we could just remove `request.Refresh` -- it kinda makes it confusing